### PR TITLE
Include keyboards when parsing messages from JSON

### PR DESCRIPTION
**Pull request by jmuia**
Keyboards should be included when parsing messages from JSON -- now they are.

*Note:* variants of the following block of code now appear in 3 places (responses, keyboards, and messages) and has potential to show up in attribution. Might want to consider creating a general function for it.
```python
if 'responses' in json:
    for response in json['responses']:
        response_type = response['type']
        response_cls = response_type_mapping.get(response_type, UnknownResponse)
        if response_cls is not UnknownResponse:
            del response['type']
        keyboard.responses.append(response_cls.from_json(response))
        response['type'] = response_type
```

### CODE REVIEW

File: kik/messages/keyboard_message.py [hunk 1]
```diff
@@ -1,6 +1,13 @@
+from kik.messages.keyboards import SuggestedResponseKeyboard, UnknownKeyboard
+
```
Comment (allawi) [inline kik/messages/keyboard_message.py]: @jmuia No whitespace

**Comment by jmuia** [comment]
@allawi

**Comment by kwaipun** [comment]
👀 👍

**Comment by jmuia** [comment]
@allawi @kwaipun Updated to include FriendPickerResponses

**Pull request merged**